website/docs/r/event_trigger.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 `mongodbatlas_event_trigger` provides a Event Trigger resource. 
 
+Note: If the `app_id` changes in the mongodbatlas_event_trigger resource, it will force a replacement and delete itself from the old Atlas App Services app if it still exists then create itself in the new  Atlas App Services app. See [Atlas Triggers](https://www.mongodb.com/docs/atlas/app-services/triggers/) to learn more.   
```

Comment:
Sorry about the late review - this looks good. My only suggestion is to add a comma right after "...if it still exists," to break up the sentence a little, but it's not an issue.